scripts/parse_existing_cohort.py

```diff
@@ -139,12 +139,12 @@ async def get_read_filenames(
         """
 
```

Comment:
This will now return m5's, and cause downstream errors. Maybe just filtering to match 
```suggestion
        relevant_extensions = ('.cram', '.fastq.gz', '.fastq', 'fq.gz', '.fq', '.bam')
```